_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,293 +846,1449 @@ If you need to brush up on how the IaC Library works, read the
+  # Fill in the region you want to use (only used for API calls) and the IDs of your AWS accounts (see the locals above)
+  aws_region     = local.aws_region
+  aws_account_id = local.accounts.security
+
+  # Prefix all resources with this name
+  name_prefix = "acme-security"
+
+  ################################
+  # Parameters for AWS Config
+  ################################
+  # Send Config logs to the common S3 bucket.
+  config_s3_bucket_name = local.config_s3_bucket_name
+
+  # Send Config logs and events to the logs account.
+  config_central_account_id = local.accounts.logs
+
+  # Do not allow objects in the Config S3 bucket to be forcefully removed during destroy operations.
+  config_force_destroy = false
+
+  # This account sends logs to the Logs account.
+  config_aggregate_config_data_in_external_account = true
+
+  # The ID of the Logs account.
+  config_central_account_id = local.accounts.logs
+
+  ################################
+  # Parameters for CloudTrail
+  ################################
+
+  # Send CloudTrail logs to the common S3 bucket.
+  cloudtrail_s3_bucket_name = local.cloudtrail_s3_bucket_name
+
+  # The CloudTrail bucket is created in the logs account, so don't create it here.
+  cloudtrail_s3_bucket_already_exists = true
+
+  # Encrypt CloudTrail logs using a common KMS key.
+  cloudtrail_kms_key_arn = local.cloudtrail_kms_key_arn
+
+  # The ARN is a key alias, not a key ID. This variable prevents a perpetual diff when using an alias.
+  cloudtrail_kms_key_arn_is_alias = true
+
+  ##################################
+  # Benchmark SNS alarms configuration
+  ##################################
+
+  # The ARN of an SNS topic for sending alarms about CIS Benchmark compliance issues.
+  # The topic exists in the logs account
+  cloudtrail_benchmark_alarm_sns_topic_arn = "arn:aws:sns:${local.aws_region}:${local.accounts.logs}:BenchmarkAlarmTopic"
+  ##################################
+  # Cross-account IAM role permissions
+  ##################################
+
+  # Create groups that allow IAM users in this account to assume roles in your other AWS accounts.
+  iam_groups_for_cross_account_access = local.cross_account_groups.cross_account_groups
+
+  # Allow these accounts to have read access to IAM groups and the public SSH keys of users in the group.
+  allow_ssh_grunt_access_from_other_account_arns = [
+    for name, id in local.accounts :
+    "arn:aws:iam::${id}:root" if name != "security"
+  ]
+
+  # A list of account root ARNs that should be able to assume the auto deploy role.
+  allow_auto_deploy_from_other_account_arns = [
+    # External CI/CD systems may use an IAM user in the security account to perform deployments.
+    "arn:aws:iam::${local.accounts.security}:root",
+
+    # The shared account contains automation and infrastructure tools, such as CI/CD systems.
+    "arn:aws:iam::${local.accounts.shared}:root",
+  ]
+  auto_deploy_permissions = [
+    "iam:GetRole",
+    "iam:GetRolePolicy",
+  ]
```

Comment:
Again, what auto deploy do we do here? Also, you're allowing access from the security account... But this _is_ the security account, so this doesn't quite make sense.